otter/controller.py

```diff
@@ -306,17 +316,25 @@ def modify_state(group_info):
 @defer.inlineCallbacks
 def modify_and_trigger(dispatcher, group, logargs, modifier, *args, **kwargs):
     """
-    Modify group state and trigger convergence after that
+    Modify group state and trigger convergence after that. Checks if group is
```

Comment:
The docstring should call out the failure mode - i.e. it no longer returns `Deferred with None`, it now returns "Deferred firing with None when the group state has been modified, and fails with TenantSuspendedError if the tenant containing the group is suspended".